### DISCUSSION TITLE
Not Sure if ive found an issue, or if im just missing a feature flag

### DISCUSSION THREAD

**Discussion by SelectSweet** [apache/datafusion#7253] [Q&A] [answered]
so I've only added datafusion to my cargo.toml, and now I'm getting this build error. I'm wondering if there's some feature flag I need to add to my cargo.toml to fix this, or if it's just some bug or issue.

```
error[E0599]: no method named `spawn_blocking` found for struct `JoinSet` in the current scope
  --> /home/pbox/.cargo/registry/src/index.crates.io-6f17d22bba15001f/datafusion-28.0.0/src/physical_plan/stream.rs:93:23
   |
93 |         self.join_set.spawn_blocking(f);
   |                       ^^^^^^^^^^^^^^ help: there is a method with a similar name: `spawn_on`

For more information about this error, try `rustc --explain E0599`.
```

**Comment by jonahgao** [comment, marked as answer]
Running `cargo update` should fix this.
`spawn_blocking` is a new interface introduced in tokio [1.28](https://github.com/tokio-rs/tokio/blob/master/tokio/CHANGELOG.md#1280-april-25th-2023), and it appears that your local version of Tokio is out of date.

**Reply by alamb** [reply]
We should probably update the DataFusion Cargo.toml file to say it requires `tokio 1.28` or above too 🤔
